RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Pinny Release Checklist
+
+Take this checklist and paste it into your release PR. Ensure steps are followed in order.
+
+- [ ] Tests pass in the terminal and in supported browsers.
+  - See [README](./README.md#browser-compatibility) for list of supported browsers.
```

Comment:
I agree that we should do automated tests first - I'm just saying that we should keep a list for anything that we definitely can't test via automation. Like right now, there's a bug in Pikabu where opening the menu after you've scrolled down on the page, the page has some visual problems - all the classes are added properly, but it looks wrong. Manual tests are needed for bugs that you can't automate easily.